pkg/cosmos/client.go

```diff
@@ -55,6 +55,56 @@ func (c *Client) PackageDescribe(name string, version string) (*Description, err
 	return &backwardCompatibleDesc, nil
 }
 
+// PackageList returns the packages installed in a cluster.
+func (c *Client) PackageList() (*[]Package, error) {
+	desc, _, err := c.cosmos.PackageList(context.TODO(), &dcos.PackageListOpts{
+		CosmosPackageListV1Request: optional.NewInterface(dcos.CosmosPackageListV1Request{})})
+	if err != nil {
+		return nil, err
+	}
+
+	var ps []Package
+	pAppNames := make(map[string]int)
+	index := 0
+	for _, p := range desc.Packages {
+		if val, ok := pAppNames[p.PackageInformation.PackageDefinition.Name]; ok {
```

Comment:
What if we have two packages installed with different path? E.g. jenkins instance per team?